changes.md

```
@@ -0,0 +1,25 @@
+# Project Changes Summary
+
+This document summarizes the changes made to implement Google Sign-in via Supabase and profile picture upload functionality.
+
+## `components/auth/auth-form.tsx`
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)